components/admin-panel/Menu.js

```diff
@@ -111,8 +112,8 @@ const Menu = ({ collective }) => {
           <OrganizationSettingsMenuLinks collective={collective} />
         </MenuGroup>
 
-        {/** General settings for other (non-host) profiles */}
-        <MenuGroup if={!isHost && !isType(collective, ORGANIZATION)}>
+        {/** General settings for other profiles */}
+        <MenuGroup if={!isType(collective, ORGANIZATION) && (!isHost || isIndividual)}>
```

Comment:
It's a bit difficult to parse, can we explain this condition in plain english in a comment?

Reply:
I simplified the conditions and updated the comments

Reply:
Nice